10up/headstartwp
# Feature/typescript project

### Description of the Change
Removes the old TS projects and makes the default project (wp-nextjs) TypeScript based.

### How to test the Change
Simply browse through the vercel preview url and ensure everything still works as expected



### Checklist:
